api/termsAndConditions/index.ts

```diff
@@ -0,0 +1,22 @@
+import * as express from 'express'
+import { config } from '../config'
+import { application } from '../config/application.config'
+import { http } from '../lib/http'
+import { getTermsAndConditionsUrl } from './termsAndConditionsUtil'
+
+export async function getTermsAndConditions(req: express.Request, res: express.Response) {
+    let errReport: any
+    try {
+        const url = getTermsAndConditionsUrl(config.services.termsAndConditions, application.idamClient)
+        const response = await http.get(url)
+        res.send(response.data)
+    } catch (error) {
+        // we get a 404 if the user has not agreed to Terms and conditions
+        errReport = {
+            apiError: error.data.message,
```

Comment:
this error should be managed through common error method

Reply:
Most errors are different, so no point in handling them the same. Also, different endpoints will want to return different errors based on different conditions.